pom.xml

```diff
@@ -64,7 +64,7 @@
 		<java.version>1.8</java.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<apollo-java.version>2.1.0</apollo-java.version>
-		<spring-boot.version>2.7.9</spring-boot.version>
+		<spring-boot.version>2.7.10</spring-boot.version>
```

Comment:
https://github.com/spring-projects/spring-boot/releases/tag/v2.7.11

spring boot 2.7.11 is released

Reply:
Updated